files/build_templates/startup_tsa_tsb.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description= STARTUP TSA-TSB SERVICE
+Requires=updategraph.service database.service
+After=updategraph.service database.service
```

Comment:
Does it make sure we are running this service before swss/syncd/teamd/bgp docker comes up  ?

Reply:
There is no dependency on these services right now.  Ensure this service starts before bgp should be sufficient right?

Reply:
Added "Before=bgp.service"